### ISSUE TITLE
(nice to have) `ros2 ai joke` sub-command support

### ISSUE THREAD

**Issue by fujitatomoya** [fujitatomoya/ros2ai#3]
basically this can be done with following implementation details.

- this command is just for fun, does nothing else...
- `-c` option, how many times do you want joke from AI? we do not want to waste the api key usage on this, so never loop forever. default 1.
- can OpenAI with `give me today's joke` and wait on the completion.
- once content comes back, it can publish that message as `str_msgs/string`.

**Comment by fujitatomoya** [comment]
no plan for this fun, closing https://github.com/fujitatomoya/ros2ai/commit/2309dae65031c065ab6f2add5623256920ff5709
